cluster/cluster.go

```diff
@@ -449,6 +452,20 @@ func parseIngressExtraVolumeMounts(ingressMap map[string]interface{}, rkeConfig
 	return nil
 }
 
+func parseIngressHostNetwork(ingressMap map[string]interface{}, rkeConfig *v3.RancherKubernetesEngineConfig) error {
+	// Setting up default behaviour so as to not catch out
```

Comment:
what's the reasoning for not setting up default values for http port as well?

Reply:
The default http and https ports are applied from ingress templates in kdm.

The validation here is more to ensure that we check for valid possibilities, and I ended up using this opportunity to apply a default if nothing is specified. 

If needed I can handle the default in the kdm ingress template as well.

Reply:
I have made the changes to move default http/https ports from templates to rke.